### ISSUE TITLE
Problem with Rhea mapping 19669 , multiple false positives

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#5492]
GO:1990606 | membrane scission GTPase motor activity | 8
-- | -- | --

guf1 |   | IEA with 19669 | GO_REF:0000116
ria1 |   | IEA with 19669 | GO_REF:0000116
sar1 |   | IEA with 19669 | GO_REF:0000116
tif213 |   | IEA with 19669 | GO_REF:0000116
tif52 |   | IEA with 19669 | GO_REF:0000116

These seem to be false positives
SPBC31F10.06c | sar1 | ADP-ribosylation factor Sar1
SPCC553.08c | ria1 | GTPase Ria1
SPAC1B3.04c | guf1 | mitochondrial elongation factor GTPase Guf1
SPBC17G9.09 | tif213 | translation initiation factor eIF2 gamma subunit
SPAC56F8.03 | tif52 | translation initiation factor eIF5B Tif52

**Comment by pgaudet** [comment]
@ValWood 

This is a broadMatch issue. We need the export to be fixed.... no point reporting these while the export is still not doing what we need it to. 

I made this high priority. 

@balhoff

**Comment by ValWood** [comment]
Yep that would be good to fix with high priority . It's causing a lot of misannotations.

I had to filter over 400 incorrect IEAs for Pombe just from these few tickets! (Although these were the only problems I spotted, most of the rest are OK—or at least don't create tens of annotations.
I checked  every mapping with >4 propagated annotation.

**Comment by balhoff** [comment]
Filtering broad matches is done now and should appear in snapshot soon.

**Comment by ValWood** [comment]
Awsome, thanks!

**Comment by ValWood** [comment]
Does this affect EC  mappings as well? I am seeing multiple EC mapping errors. Should I report those?

**Comment by pgaudet** [comment]
yes, EC is also affected - mapping types were being ignored on export, both in the .obo ontology files and in the mappings files (https://release.geneontology.org/2024-09-08/ontology/external2go/index.html) 

Which one are you using? 

Anyways, this should be fixed very soon. 

Thanks, Pascale

**Comment by ValWood** [comment]
We don't make the IEA annotations ourselves we get them from GOA, so will need to wait for the fixes to filter through

the ones I just spotted are lots of mappings to different polymerases from 
https://www.pombase.org/gene/SPBC28F2.12 rpb1

**Comment by pgaudet** [comment]
OK. It should be fixed soon ...
